app/services/api.rb

```diff
@@ -70,4 +74,9 @@ def self.writable_model?(model)
     @writable ||= writable_models.map { |m| m.name.underscore }
     @writable.include?(model.to_s.underscore.singularize)
   end
+
+  def self.publishable_model?(model)
```

Comment:
I forgot it is also used for setting constraints on routes so I am leaving it as is.